portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,16 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomProject() {
+  const Projects =
+      ['Porject Title - Big Integer for c++ \nShort Description\nBuilt a c++ Library to handle integers greater than 64 bit.'
+	  , 'Porject Title - Text encryptor/decryptor \nShort Description\nBuilt a text encryptor/decryptor program using basic ciphers like vigenere cipher and rail fence cipher.'
```

Comment:
Comma should be after the last element. There shouldn't be any space between element and comma. The next element can begin on a new line or after a space.